src/Pmi/Console/Command/DeployCommand.php

```diff
@@ -388,20 +386,13 @@ private function displayVulnerabilities($vulnerabilities)
 
     private function runSecurityCheck()
     {
-        $composerLockFile = $this->appDir . DIRECTORY_SEPARATOR . 'composer.lock';
-        $this->out->writeln("Running SensioLabs Security Checker...");
-        $checker = new SecurityChecker();
-        $helper = $this->getHelper('question');
-        try {
-            $vulnerabilities = json_decode((string)$checker->check($composerLockFile), true);
-        } catch (HttpException $e) {
-            $this->out->writeln('<error>' . $e->getMessage() . '</error>');
-            if (!$helper->ask($this->in, $this->out, new ConfirmationQuestion('Continue anyways? '))) {
-                throw new \Exception('Aborting due to SensioLabs Security Checker network error');
-            }
-        }
-        // Ignore vulnerabilities mentioned in sensioignore file
-        $vulnerabilities = $this->removeSensioIgnoredVulnerabilities($vulnerabilities);
+        $this->out->writeln("Running Symfony Security Check...");
+        $process = new Process('symfony security:check --format=json --dir=' . escapeshellarg($this->appDir));
```

Comment:
Only thing I might suggest here is running a `which symfony` check and throwing a console error if it evaluates false, prompting the developer to install it (or fix their `$PATH`).